cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -454,6 +457,15 @@ public boolean verify(String s, SSLSession sslSession) {
                 args = args.subList(2,args.size());
                 continue;
             }
+            if(head.equals("-v")) {
```

Comment:
you may expose numeric value of logging level and note in docs what level is for what value